docs/archive/api/sql/GeoSparkSQL-Constructor.md

```diff
@@ -155,6 +155,24 @@ FROM polygontable
 SELECT ST_PolygonFromText('-74.0428197,40.6867969,-74.0421975,40.6921336,-74.0508020,40.6912794,-74.0428197,40.6867969', ',') AS polygonshape
 ```
 
+## ST_LineFromText
```

Comment:
This doc is put in wrong place

Reply:
I have moved below ST_GeomFromWKT. Let me know if it has to be moved to some other place or file.